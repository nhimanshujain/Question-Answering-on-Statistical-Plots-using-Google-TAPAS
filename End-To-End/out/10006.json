{
    "frame": "10006",
    "scores": [
        0.9999945163726807,
        0.9999923706054688,
        0.9999701976776123,
        0.9999642372131348,
        0.9999575614929199,
        0.9999561309814453,
        0.9999512434005737,
        0.9999502897262573,
        0.9999496936798096,
        0.9999446868896484,
        0.9999395608901978,
        0.9999374151229858,
        0.999933123588562,
        0.9999309778213501,
        0.9999247789382935,
        0.9999061822891235,
        0.9998942613601685,
        0.9998867511749268,
        0.9998866319656372,
        0.9998660087585449,
        0.9996670484542847,
        0.9995629191398621
    ],
    "categories": [
        9,
        9,
        2,
        0,
        7,
        0,
        0,
        0,
        0,
        2,
        2,
        4,
        4,
        0,
        7,
        7,
        6,
        7,
        7,
        4,
        5,
        8
    ],
    "bboxes": [
        [
            27.25185203552246,
            459.97637939453125,
            63.65839385986328,
            474.07421875
        ],
        [
            40.192203521728516,
            102.92390441894531,
            64.24410247802734,
            117.1244888305664
        ],
        [
            1173.8287353515625,
            619.1683349609375,
            1215.5662841796875,
            639.175048828125
        ],
        [
            74.69889831542969,
            109.78387451171875,
            726.3924560546875,
            163.60403442382812
        ],
        [
            735.3030395507812,
            612.0874633789062,
            747.5630493164062,
            625.9378662109375
        ],
        [
            79.21051025390625,
            521.163330078125,
            1080.1463623046875,
            574.4030151367188
        ],
        [
            75.50068664550781,
            467.32183837890625,
            653.8601684570312,
            521.538818359375
        ],
        [
            79.16535949707031,
            56.312198638916016,
            898.49658203125,
            110.08055877685547
        ],
        [
            76.8082275390625,
            414.31732177734375,
            815.82861328125,
            467.9610595703125
        ],
        [
            1174.101806640625,
            596.1893920898438,
            1235.420166015625,
            616.2551879882812
        ],
        [
            1173.954833984375,
            572.9148559570312,
            1208.17236328125,
            592.91943359375
        ],
        [
            1148.673583984375,
            572.9439697265625,
            1168.8115234375,
            593.0646362304688
        ],
        [
            1148.6539306640625,
            619.1229858398438,
            1168.7286376953125,
            638.9912109375
        ],
        [
            76.31050872802734,
            164.4281768798828,
            1101.3408203125,
            217.30909729003906
        ],
        [
            512.8422241210938,
            611.9534912109375,
            524.9588012695312,
            625.9398803710938
        ],
        [
            290.6319580078125,
            612.0994262695312,
            303.05084228515625,
            626.0181274414062
        ],
        [
            546.7985229492188,
            629.4925537109375,
            666.7786865234375,
            647.783447265625
        ],
        [
            72.15039825439453,
            612.00048828125,
            78.21342468261719,
            625.9788818359375
        ],
        [
            956.0477905273438,
            612.1361083984375,
            968.4012451171875,
            626.0216064453125
        ],
        [
            1149.1019287109375,
            595.9469604492188,
            1169.0247802734375,
            616.4177856445312
        ],
        [
            76.6021499633789,
            9.004288673400879,
            769.38330078125,
            27.01488494873047
        ],
        [
            6.17329740524292,
            301.8087463378906,
            24.040910720825195,
            330.2987976074219
        ]
    ],
    "total_instances": 22
}