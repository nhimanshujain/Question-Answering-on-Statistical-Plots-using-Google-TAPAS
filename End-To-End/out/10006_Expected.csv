Sex,Algen'a,Morocco,Cote d'Ivoire,Unknown,xlabel,ylabel,title,legend orientation
Female,94.32879505594684,66.39639456793714,52.37143116419264,,Survival Rate(%),Sex,Survival rate of children of different countries in Grade 5 of primary education,vertical
Male,95.32712755875369,73.50558381424864,58.67107965988478,,Survival Rate(%),Sex,Survival rate of children of different countries in Grade 5 of primary education,vertical
